plugin-server/src/cdp/consumers/cdp-events.consumer.ts

```diff
@@ -167,6 +168,60 @@ export class CdpEventsConsumer extends CdpConsumerBase {
                 'hog_function'
             )
 
+            const eventsTriggeredDestinationById: Record<string, { team_id: number; event_uuid: string }> = {}
+            const uniqueDestinationsById: Record<
+                string,
+                { team_id: number; template_id: string; invocation_id: string }
+            > = {}
+
+            notMaskedInvocations.forEach(({ state, hogFunction, id }) => {
+                const eventKey = `${state.globals.project.id}:${state.globals.event.uuid}`
+                if (!eventsTriggeredDestinationById[eventKey] && hogFunction.type === 'destination') {
+                    eventsTriggeredDestinationById[eventKey] = {
+                        team_id: state.globals.project.id,
+                        event_uuid: state.globals.event.uuid,
+                    }
+                }
+
+                const destinationKey = `${state.globals.project.id}:${id}`
+                if (!uniqueDestinationsById[destinationKey] && hogFunction.type === 'destination') {
+                    uniqueDestinationsById[destinationKey] = {
+                        team_id: state.globals.project.id,
+                        template_id: hogFunction.template_id ?? 'custom',
+                        invocation_id: id,
+                    }
+                }
+            })
+
+            const uniqueEventMetrics: MinimalAppMetric[] = Object.entries(eventsTriggeredDestinationById).map(
+                ([_, { team_id, event_uuid }]) => {
```

Comment:
You don't use the key so rather Object.values
```suggestion
            const uniqueEventMetrics: MinimalAppMetric[] = Object.values(eventsTriggeredDestinationById).map(
                ({ team_id, event_uuid }) => {
```